src/data/multiset/basic.lean

```diff
@@ -2147,6 +2147,18 @@ begin
     simp only [h, if_false, zero_min],
 end
 
+/-- `multiset.map f` preserves `count` if `f` is injective on the set of elements contained in
+the multiset -/
+theorem count_map_eq_count [decidable_eq β] (f : α → β) (s : multiset α)
```

Comment:
... and this is a possible name change in consequence
```suggestion
theorem _root_.set.inj_on.count_map_multiset [decidable_eq β] (f : α → β) (s : multiset α)
```

Reply:
I'm not sure this makes sense. It seems a pretty unlikely use case for dot notation, and the price of putting the lemma in an unexpected namespace (plus the minor ugliness of having to write `_root_` here).